src/k8s/mock_templates/fedora28.mock.js

```diff
@@ -3,7 +3,7 @@ export const fedora28 = {
   kind: 'Template',
   metadata: {
     name: 'fedora-generic',
-    namespace: 'default',
+    namespace: 'openshift',
```

Comment:
common-templates are installed into openshift namespace so I want to use the same namespace here too.